align/pnr/build_pnr_model.py

```diff
@@ -157,9 +157,8 @@ def PnRdatabase( path, topcell, vname, lefname, mapname, drname):
     else:
         global_signals = DB.ReadVerilog( path, vname, topcell)
 
-    _attach_constraint_files( DB, path)
     DB.semantic0( topcell)
     DB.semantic1( global_signals)
     DB.semantic2()
-
+    _attach_constraint_files( DB, path)
```

Comment:
@stevenmburns We are adding some constraints related to power net. Is it OK to read constraints after semantic? Because power net is built in semantic.